client/blocks/credit-card-form/index.jsx

```diff
@@ -172,7 +172,12 @@ const CreditCardForm = React.createClass( {
 						this.setState( { formSubmitting: false } );
 					}
 
-					notices.error( message );
+					if ( typeof message === 'object' ) {
+						notices.error( <ValidationErrorList messages={ message } /> );
```

Comment:
We get React warning:
![screen shot 2016-09-01 at 12 34 18](https://cloud.githubusercontent.com/assets/699132/18164505/7a06425e-7040-11e6-8a8b-5eee178e2dd9.png)

It looks like we need to pass array of messages in this place.